libs/ui/src/reboot_from_usb_button.test.tsx

```diff
@@ -48,32 +46,28 @@ test('renders with a non-bootable USB as expected', async () => {
       logger={new Logger(LogSource.VxAdminFrontend)}
     />
   );
-  await act(
-    async () => await fireEvent.click(screen.getByText('Reboot from USB'))
-  );
+  await act(async () => fireEvent.click(screen.getByText('Reboot from USB')));
```

Comment:
is there a way to detect unnecessary `async` functions as well? maybe `@typescript-eslint/require-await`

Reply:
I'll see about enabling that.